examples/cra-kitchen-sink/.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": ["env", "stage-0", "react"],
+  "plugins": [
+    ["transform-runtime", {
+      "polyfill": false,
+      "regenerator": true
+    }],
+	"react-docgen"
```

Comment:
I don't think it's a good idea to add this `.babelrc` since these settings should have been the default included by Storybook.

If there's anything that should be inserted, we should include it here:
https://github.com/storybooks/storybook/blob/master/app/react/src/server/babel_config.js#L73

Looks like the babel plugin is already getting inserted.